tests/Unit/WaitsForElementsTest.php

```diff
@@ -53,6 +55,55 @@ public function test_when_available()
         }
     }
 
+    public function test_when_available_missing()
```

Comment:
These 2 tests reproduce the 2 use cases in `test_when_available()`, when the item is found and when it's not found.
Although these new tests have some duplication in the mock setup, they are easier to understand, which I think is better, considering the inherent complexity of testing a method that uses a callback